### ISSUE TITLE
「Feature」

### ISSUE THREAD

**Issue by Fan-Bob** [zhanghai/MaterialFiles#1222]
![2](https://github.com/zhanghai/MaterialFiles/assets/139612414/0209b19b-9968-411f-8896-db418f0b1000)
I hope I can change the order of lists, pictures, music, etc. on the left.

**Comment by LondoJI** [comment]
You're right. Seems to be like they are set alphabetically by default. We can turn them on/off, but not change their positions as we are already free to do with our bookmarks.

**Comment by Feuerswut** [comment]
@author please add a more specific title for the issue
